helm/templates/storage/README.md

```diff
@@ -2,9 +2,13 @@
 
 The `storage` Helm chart installs the SBOMbastic storage deployment, which should be installed alongside the SBOMbastic controller and worker components.
 
-The storage component uses SQLite as its database backend. **Note that SQLite is intended for development and testing purposes only, and should not be used in production environments.** 
+The storage component uses SQLite as its database backend. **Note that SQLite is intended for development and testing purposes only, and should not be used in production environments.**
 
-To ensure data persistence, it requires a PersistentVolumeClaim (PVC) backed by a PersistentVolume (PV). Users need to either:
+To ensure data persistence, the storage component requires a PersistentVolumeClaim (PVC). You can provide your own PVC to control how and where data is stored. This is the recommended approach and aligns with common Helm chart practices.
 
-1. Have a StorageClass configured that can dynamically provision PVs, or
-2. Manually create a PV that matches the PVC requirements
\ No newline at end of file
+There are two ways to satisfy this requirement:
+
+1. **Recommended:** Provide a pre-created PVC and reference it in your Helm values using `persistence.storageData.existingClaim`.
+2. If no PVC is provided, and your cluster supports dynamic provisioning via a StorageClass, a new PVC and corresponding PV will be created automatically.
```

Comment:
Why the 1st solution is the recommended one? I personally think the 2nd one is the simplest one, since it's all "self-service".


```suggestion
1. Provide a pre-created PVC and reference it in your Helm values using `persistence.storageData.existingClaim`.
2. If no PVC is provided, and your cluster supports dynamic provisioning via a `StorageClass`, a new PVC and corresponding PV will be created automatically.
```